src/components/Editable.tsx

```diff
@@ -388,6 +389,8 @@ const Editable = ({ disabled, isEditing, simplePath, path, cursorOffset, showCon
       }
     }
 
+    isTapped && setIsTapped(false)
```

Comment:
Let's stick with explicit `if` statements on expressions with side effects. I'm not strictly opposed to the shorthand, but it deviates from the convention we have been using so I'd like to stay consistent for now.